deps/net.lua

```diff
@@ -154,6 +154,16 @@ function Socket:keepalive(enable, delay)
   uv.tcp_keepalive(self._handle, enable, delay)
 end
 
+function Socket:buffer_size(send_size, recv_size)
```

Comment:
I think this should be called `Socket:set_buffer_size` and then a `Socket:get_buffer_size` should be added too:

```lua
function Socket:get_buffer_size()
  local send_size = uv.send_buffer_size(self._handle)
  local recv_size = uv.recv_buffer_size(self.handle)
  return send_size, recv_size
end
```

Reply:
Agree, but it should be camelCase.